docs: Update sql docs (#8)
CeresDB/docs: docs/src/sql/ddl/drop_table.md

# CREATE TABLE

## Basic syntax

> Basic syntax (parts between `[]` are optional):

Reviewer: ```suggestion
Basic syntax:
```

```sql

> DROP TABLE [IF EXIST] table_name [,tb1_name]

Reviewer: I guess we don't support drop multiple tables in one sql:
```suggestion
DROP TABLE [IF EXIST] table_name
```

```

> `Drop Table` removes one or more tables.You should be careful to use this statement, it removes the table definition and table data which is not recoverable.

Reviewer: ```suggestion

`Drop Table` removes a specific table. This statement should be used with caution, because it removes both the table definition and table data, and this removal is not recoverable.
```
